spaceship.zsh

```diff
@@ -52,6 +52,7 @@ SPACESHIP_SWIFT_SYMBOL="${SPACESHIP_SWIFT_SYMBOL:-🐦}"
 
 # GOLANG
 SPACESHIP_GOLANG_SHOW="${SPACESHIP_GOLANG_SHOW:-true}"
+SPACESHIP_GOLANG_SHOW_ALWAYS="${SPACESHIP_GOLANG_SHOW_ALWAYS:-false}"
```

Comment:
I'd rather remove this option. [We have already thought about this](https://github.com/denysdovhan/spaceship-zsh-theme/pull/28#issuecomment-242972455) in #28. We will add this in future.